device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsTelemetrySenderLinkHandler.java

```diff
@@ -28,7 +28,7 @@
 
     AmqpsTelemetrySenderLinkHandler(Sender sender, AmqpsLinkStateCallback amqpsLinkStateCallback, DeviceClientConfig deviceClientConfig, String linkCorrelationId)
     {
-        super(sender, amqpsLinkStateCallback, linkCorrelationId);
+        super(sender, amqpsLinkStateCallback, linkCorrelationId, deviceClientConfig.getModelId());
```

Comment:
Does that mean deviceClientConfig will never be null? do we need a nullcheck?

Reply:
It won't be null here, no